istio-proto/src/main/protobuf/google/rpc/README.md

```diff
@@ -0,0 +1,5 @@
+# Google RPC
```

Comment:
The "g" in gRPC apparently [no longer stands for Google](https://grpc.io/faq/) :)

Reply:
This was copied verbatim from https://github.com/googleapis/googleapis/blob/master/google/rpc/README.md#google-rpc so I think we should keep it but... it had actually never occurred to me that the `g` stood for "Google", I always thought it was something like "generalised" :P